lavamoat/webpack/policy-override.json

```diff
@@ -173,15 +135,12 @@
     },
     "eth-lattice-keyring>gridplus-sdk>buffer": {
       "globals": {
-        "SharedArrayBuffer": true,
-        "SharedArrayBuffer.Symbol(Symbol.hasInstance)": true,
-        "SharedArrayBuffer.Symbol(Symbol.hasInstance).name": true
+        "SharedArrayBuffer": true
       }
     },
     "@open-rpc/test-coverage>isomorphic-fetch>whatwg-fetch": {
       "globals": {
         "DOMException": true,
```

Comment:
it seems like this package is replacing `DOMException.prototype` and we're allowing it if we give it the entirety of `DOMException`. Hope that's fine.